classes/Language.php

```diff
@@ -644,7 +644,7 @@ public function deleteSelection($selection)
      * @see loadLanguages()
      *
      * @param bool $active Select only active languages
-     * @param int|false $id_shop Shop ID
+     * @param int|bool $id_shop Shop ID
```

Comment:
So PHPStan doesn't accept `false` as PHPDoc type? That's a problem for my previous comments.. 🤔

Reply:
true & false  are not in primitive type for PHPDoc : https://docs.phpdoc.org/3.0/guide/guides/types.html